easybuild/easyconfigs/s/SlurmViewer/SlurmViewer-1.0.2-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,61 @@
+# #
+# Authors:    
+#  - Patrick de Koning <p.j.h.de_koning@lumc.nl>
+#  - Prerak Mody <prerakmody@gmail.com>
+# #
+easyblock = 'PythonBundle'
+
+name = 'SlurmViewer'
+version = '1.0.2'
+
+homepage = 'https://gitlab.com/lkeb/slurm_viewer'
+description = """One-stop command to view the status of a Slurm cluster, including nodes, jobs and resource utilization"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('hatchling', '1.18.0'),
+    ('poetry', '1.6.1'),
+]
+
+dependencies = [
+    ('Python', '3.11.5'),
+    ('Python-bundle-PyPI', '2023.10'),
+    ('pydantic', '2.7.4'),
```

Comment:
```suggestion
    ('pydantic', '2.9.1'),
```

Reply:
Tested with pydantic == 2.9.1 and it works.